pro/scripts/customRequest.ts

```diff
@@ -252,10 +252,24 @@ const getResponseBody = async (response: Response): Promise<any> => {
   if (response.status !== 204) {
     try {
       const contentType = response.headers.get('Content-Type')
+
       if (contentType) {
-        const isJSON = contentType.toLowerCase().startsWith('application/json')
+        const lowercaseContentType = contentType.toLowerCase()
+        const isJSON = lowercaseContentType.startsWith('application/json')
+
+        // Get around this issue
```

Comment:
commentaire peut clair, this issue fait référence à quelle issue ?

Reply:
j'ai oublié de mettre le lien :facepalm: